perun-core/src/main/resources/oracleChangelog.txt

```diff
@@ -6,6 +6,60 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.1.47
+delete from service_dependencies where dependency_id in (select id from exec_services where exec_services.type='GENERATE');
+delete from service_denials where exec_service_id in (select id from exec_services where exec_services.type='GENERATE');
+delete from tasks where exec_service_id in (select id from exec_services where exec_services.type='GENERATE');
+delete from exec_services where type='GENERATE';
+alter table tasks drop constraint TASK_STAT_CHK;
+update tasks set status='ERROR' where status != 'ERROR' and status != 'DONE';
+alter table tasks add constraint TASK_STAT_CHK check (status in ('WAITING', 'PLANNED', 'GENERATING', 'GENERROR', 'GENERATED', 'SENDING', 'DONE', 'SENDERROR', 'ERROR'));
+alter table services add column description nvarchar2(1024);
+alter table services add column delay integer default 10 not null;
+alter table services add column recurrence integer default 2 not null;
+alter table services add column enabled char(1) default '1' not null;
+alter table services add column script nvarchar2(256);
+update services set script=exec_services.script from exec_services where services.id=exec_services.service_id;
```

Comment:
Please fix setting column values from select like this:
```sql
update services set script=(select exec_services.script from exec_services where services.id=exec_services.service_id);
```
Same pattern for line below.